app/src/androidMain/kotlin/de/westnordost/streetcomplete/quests/opening_hours/AddOpeningHours.kt

```diff
@@ -131,7 +131,7 @@ mapOf(
                     or amenity = recycling and recycling_type = centre
                     or amenity = toilets and (fee = yes or toilets:disposal = flush)
                     or amenity = shower and (fee = yes or indoor = yes or location = indoor)
-                    or tourism = information and information = office
+                    or tourism = information and information ~ office|visitor_center
```

Comment:
Typo: OSM usually uses English spelling, so this should be


```suggestion
                    or tourism = information and information ~ office|visitor_centre
```

see https://taginfo.openstreetmap.org/tags/information=visitor_centre

Reply:
Ok wat?
So I tested adding a "visitor center" and the quests worked. But it seems like id-tagging-schema just classifies visitor_center as office: https://github.com/openstreetmap/id-tagging-schema/blob/d87f48d201a0cc49988fffeebaf4e026a841b596/data/presets/tourism/information/office.json#L37-L38

Good find though, thanks!